src/components/media/Media.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+const Media = ({icon, label}) => {
+    return(
+        <div className="media">
+            <span className="media__icon">{icon}</span>
+            <span className="media__body">{label}</span>
+        </div>
+    )
+}
+
+export default Media;
```

Comment:
Please add a new line at the end of your code :)

Reply:
🆗